manimpango/cmanimpango.pyx

```diff
@@ -138,22 +140,28 @@ class MarkupUtils:
         slant: str,
         weight: str,
         size: int,
-        line_spacing: int,
+        _: int, # for some there was a keyword here.
         disable_liga: bool,
         file_name: str,
         START_X: int,
         START_Y: int,
         width: int,
         height: int,
+        *, # keyword only arguments below
+        justify: bool = None,
+        indent: float = None,
+        line_spacing: float = None,
+        alignment: Alignment = None
     ) -> int:
         """Render an SVG file from a :class:`manim.mobject.svg.text_mobject.MarkupText` object."""
         cdef cairo_surface_t* surface
         cdef cairo_t* context
         cdef PangoFontDescription* font_desc
         cdef PangoLayout* layout
         cdef cairo_status_t status
-        cdef double width_layout = width
+        cdef double width_layout = width - 50
```

Comment:
using justify made the text render outside the SVG which hide some texts. Doing so fixed it.